controller/Review.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+ *
+ *
+ */
+
+namespace oat\taoReview\controller;
+
+use common_exception_Error;
+use oat\ltiDeliveryProvider\model\LtiLaunchDataService;
+use oat\ltiDeliveryProvider\model\LtiResultAliasStorage;
+use oat\taoLti\models\classes\LtiException;
+use oat\taoLti\models\classes\LtiService;
+use oat\taoLti\models\classes\LtiVariableMissingException;
+use tao_actions_SinglePageModule;
+
+/**
+ * Review controller class thar provides data for js-application
+ * @package oat\taoReview\controller
+ */
+class Review extends tao_actions_SinglePageModule
+{
+    /**
+     * @throws LtiException
+     * @throws LtiVariableMissingException
+     * @throws common_exception_Error
+     */
+    public function index()
+    {
```

Comment:
to much logick for a controller. it should be separated to the service

Reply:
Let's create a follow-up task for the next sprint for this